pynestml/codegeneration/nest_code_generator.py

```diff
@@ -322,8 +323,7 @@ def analyse_neuron(self, neuron: ASTModel) -> Tuple[Dict[str, ASTAssignment], Di
         equations_block = neuron.get_equations_blocks()[0]
 
         kernel_buffers = ASTUtils.generate_kernel_buffers(neuron, equations_block)
-        ASTUtils.make_inline_expressions_self_contained(equations_block.get_inline_expressions())
-        ASTUtils.replace_inline_expressions_through_defining_expressions(equations_block.get_ode_equations(), equations_block.get_inline_expressions())
+        InlineExpressionExpansionTransformer().transform(neuron)
```

Comment:
Could this be moved to `pynestml_frontend.py` where all the other transformers are called?

Reply:
I agree that this would be nice! Unfortunately, I tested it, and it does not work due to the order of transformations---the inline expansion currently takes place at a specific stage in the code generator. I think we might be able to do as you suggested after completing https://github.com/nest/nestml/pull/1050, but that will probably still be some time into the future.